deployment.rb

```diff
@@ -70,7 +70,9 @@ def load_configuration()
     'ruby_installer'              => rack_env == :development ? 'rbenv' : 'system',
     'root_dir'                    => root_dir,
     'use_dynamo_tables'           => false,#[:staging, :production].include?(rack_env),
-    'dynamo_table_name'           => "#{rack_env}_tables",
+    'dynamo_tables_table'           => "#{rack_env}_tables",
+    'use_dynamo_properties'       => false,
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.